src/audio/eq_iir/eq_iir.c

```diff
@@ -37,9 +37,7 @@
 
 LOG_MODULE_REGISTER(eq_iir, CONFIG_SOF_LOG_LEVEL);
 
-/* 5150c0e6-27f9-4ec8-8351-c705b642d12f */
-SOF_DEFINE_UUID("eq_iir", eq_iir_uuid, 0x5150c0e6, 0x27f9, 0x4ec8,
-		 0x83, 0x51, 0xc7, 0x05, 0xb6, 0x42, 0xd1, 0x2f);
+SOF_DEFINE_REG_UUID(eq_iir);
```

Comment:
can we also automate LLEXT UUUID uses? https://github.com/thesofproject/sof/blob/d51fa68079aed2dcb8f27f6e2eaffe15c79e7850/src/audio/eq_iir/eq_iir.c#L266

Reply:
Didn't notice this one.  Will do.

Reply:
@andyross is this planned for a future PR?